docs/.custom_wordlist.txt

```diff
@@ -1,6 +1,7 @@
 CLI
 ESM
 Geneve
+Golang
```

Comment:
Looking at Go documentation and product pages, I do not actually see them use this way of presenting themselves, I just see the use of `Go`. Should we perhaps stick to that too?

Reply:
you are right. This is probably just my bad habit.

Reply:
`Golang` removed in favor of `Go`